dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1662,7 +1662,7 @@ def check_compression(engine, filename, compression):
         else:
             assert md.total_compressed_size != md.total_uncompressed_size
     else:
-        metadata = pa.parquet.ParquetDataset(filename).metadata
+        metadata = pa.parquet.read_metadata(filename + "/_metadata")
```

Comment:
I thought we would have needed to use `os.path.join(...)` here for this to work on Windows, but I guess not...It looks like tests that use `check_compression` are passing in our Windows builds.

Reply:
But let's maybe make it more robust using os.path.join anyway.